packages/login/src/views/StepOne/StepOneForm.tsx

```diff
@@ -92,7 +97,7 @@ export const StyledPrimaryButton = styled(PrimaryButton)`
 `
 
 export const StyledButton = styled(LinkButton)`
```

Comment:
Update can be updated to use < Link /> which can also show the loading state

Reply:
alright..